account.go

```diff
@@ -224,11 +225,23 @@ func (account *account) Transact(ctx context.Context, preConditionCheck func() b
 			continue
```

Comment:
This continue should be removed. Regardless of errors, check the post condition.

Reply:
Fixed